### ISSUE TITLE
Composite attribute template not being applied to pk in a query with sort key operations.

### ISSUE THREAD

**Issue by jamesgibbons92** [tywalch/electrodb#225]
**Describe the bug**
Composite attribute template is not being applied to pk in params when executing a query with sort key operations.
We are trying to support existing data by specifying pk.template in our entities, the pk is a uuid style id with an entity type appended. e.g. `13cc114a-0fe1-4d7c-8e30-0c6853ca646c-notification`
Our pk.template is specified as `'${userId}-notification'`
When applying sort key operator to a query this template is not being applied to the pk and it is being constructed as `${userId}`, the value sent in the resulting dynamo request params is therefore incorrect.
When querying without a sort key operator, or carrying out other query/mutation operations this behavior is not observed and the pk correctly has -notification appended to the ID.

This behavior can be observed quite clearly in the playground link below.

**ElectroDB Version**
 `2.4.2`

**ElectroDB Playground Link**
[ElectroDB Playground](https://electrodb.fun/#code/PQKgBAsg9gJgpgGzARwK5wE4Es4GcA0YuccYGeqCALgUQBYCG5YA7llXWAGbZwB2MXGBDAAUKKwBbAA5QMVMAG8wAUT5V2AT0IBlTADcsAY1IBfbhiiSwAIkRwjVSzABGNgNzijUPrgVUGFwRSAF5bTShUDAB9AKC4aL4GSTgPcVAwADkoDS5jBg0fVXUtYTFvXwVs3PzC3zAwvjgWYo0qTQAKUTAlbp6wSVhEAC5e-v7+Ns1RgHI+HKw8owKsHxn8PvH9TFxVvlmARnXN-uIMQxNZ+Zrlutxj8dMN8YKnLBdUKjxRxROe1DOAEkYD8-v12tI4LM-Ng+ABzB7jMBPMEpXC4BhwqFjJE9CHYmYwrDwxGPZ5IjQpUG4vGaSHQt4k8lkk4o8bE+AAD2+OPGLk0AFUgSDeUjpABrak0np5RAimYAzDA0k0r4yBAFAkAEkUiowwNMAFprotansVbjvDIoLsvqMANoK4UzAC6zKRbJpuElotxsoQ8vIaDwVGV7txaukGrtYBmOspcFMFqRVtktuxjoTrvDKYYu3hs1Q0khGGTPU9LMryPJv3GcWC5JRAEp0sAwHBOZDHHAYGBDWAJWAsEIXBgGDBDQAGA7GhZLFY+UTVU23Pa4AB0wYwmhO6-5QqVMA6taReuBswAQmOJ9OVaYW+N13CoB0m55RMA26gmp2HF9e-2g7DmAo7jlOBxLnOZo+BuW47o++7CseYJnvKV5gbe7r3rucJfMeCajDYABMk5EUR4HgQAKpOk7DDRdGTuuNGTgAWjY2GPs+r7vp+7a-t2AEDuKQ4jteFEmvOdSQTcC6+OuRjkJqyH-MKl5iZhfQEbGJEkRR07UbR9E0UxNEsYiaIYlihEABJYDYoj3k+L4th+bYdl2-59kJIkgepM4SdBfDSSuskbkWMBKSeqFqRhRzklpMw6ZOekHAZDEMSZrEzA5TbrsQVDKQMeCWdiNjWYgCBQPZjlcS5vHuX+PZeUBomxbOMlScuklrk+cAFSeWk2AcREAMw2OS0W2OhN4HNVuW1UAA) 

**Entity/Service Definitions**
```
const Notifications = new Entity(
  {
    model: {
      entity: 'notification',
      version: '1',
      service: 'notifications',
    },
    attributes: {
      userId: {
        type: 'string',
      },
      message: {
        type: 'string',
      },
      time: {
        type: 'string',
      },
    },
    indexes: {
      byUserId: {
        pk: {
          field: 'userId',
          template: '${userId}-notification',
          composite: ['userId'],
        },
        sk: {
          field: 'requestId',
          template: '${time}',
          composite: ['time'],
          casing: 'upper',
        },
      },
    },
  },
  {
    table,
  },
)
```

**Comment by tywalch** [comment]
Thank you make putting this together, I'll be looking into this 👍

**Comment by tywalch** [comment]
@jamesgibbons92 This was fixed in version `2.6.0` 👍
